sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/linkedin/CollectionResourceManager.java

```diff
@@ -98,11 +99,15 @@ private void applyDesiredContainerPolicies(final CosmosAsyncContainer container,
     private void createContainer(final CosmosAsyncDatabase database,
         final String containerName,
         final CollectionAttributes collectionAttributes) {
-        LOGGER.info("Creating container {} in the database {}", containerName, database.getId());
+        LOGGER.info("Creating container {} in the database {} [throughput = {}]", containerName,
+            database.getId(),
+            _configuration.getThroughput());
+        final ThroughputProperties throughputProperties =
+            ThroughputProperties.createManualThroughput(_configuration.getThroughput());
         final CosmosContainerProperties containerProperties =
             new CosmosContainerProperties(containerName, Constants.PARTITION_KEY_PATH)
                 .setIndexingPolicy(collectionAttributes.indexingPolicy());
-        database.createContainerIfNotExists(containerProperties)
+        database.createContainerIfNotExists(containerProperties, throughputProperties)
```

Comment:
Also lets to remove throughputProperties from DatabaseResourceManager

Reply:
That line is not setting the throughput on the database, it's using this method for creating the container in CosmosAsyncDatabase

    /**
     * Creates a Cosmos container if it does not exist on the service.
     * <p>
     * The throughput properties will only be used if the specified container
     * does not exist and therefor a new container will be created.
     *
     * After subscription the operation will be performed. The {@link Mono} upon
     * successful completion will contain a cosmos container response with the
     * created or existing container. In case of failure the {@link Mono} will
     * error.
     *
     * @param containerProperties the container properties.
     * @param throughputProperties the throughput properties for the container.
     * @return a {@link Mono} containing the cosmos container response with the
     * created or existing container or an error.
     */
    public Mono<CosmosContainerResponse> createContainerIfNotExists(

Reply:
And the line is removed from the database manager: sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/linkedin/DatabaseResourceManager.java